src/EditorFeatures/VisualBasic/NavigationBar/GenerateMethodItem.vb

```diff
@@ -27,7 +27,10 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.NavigationBar
                 Return Nothing
             End If
 
-            Dim codeGenerationSymbol = GeneratedSymbolAnnotation.AddAnnotationToSymbol(CodeGenerationSymbolFactory.CreateMethodSymbol(methodToReplicate))
+            Dim codeGenerationSymbol = GeneratedSymbolAnnotation.AddAnnotationToSymbol(
+                CodeGenerationSymbolFactory.CreateMethodSymbol(
+                    methodToReplicate,
+                    parameters:=methodToReplicate.Parameters.WithAttributesToBeCopied(destinationType)))
```

Comment:
This is untested as you can't actually hit it -- the only time we generate methods is in the partial method case, thus attributes would always be accessible. I'm fixing this out of completeness so if/when the attribute copying policy is changed, or this code gets used for something else, we don't have a bug.